utility/acquire_nex_unique_id_with_password.go

```diff
@@ -17,20 +15,15 @@ func (commonProtocol *CommonProtocol) acquireNexUniqueIDWithPassword(err error,
 	connection := packet.Sender()
 	endpoint := connection.Endpoint()
 
-	pNexUniqueIDInfo := utility_types.NewUniqueIDInfo()
-
-	uniqueId, password, nexError := commonProtocol.manager.GenerateNEXUniqueIDWithPassword(commonProtocol.manager, packet)
+	pUniqueIDInfo, nexError := commonProtocol.manager.GenerateNEXUniqueIDWithPassword(commonProtocol.manager, packet.Sender().PID())
 	if nexError != nil {
 		common_globals.Logger.Error(nexError.Error())
 		return nil, nexError
 	}
 
-	pNexUniqueIDInfo.NEXUniqueID = types.UInt64(uniqueId)
-	pNexUniqueIDInfo.NEXUniqueIDPassword = types.UInt64(password)
-
 	rmcResponseStream := nex.NewByteStreamOut(endpoint.LibraryVersions(), endpoint.ByteStreamSettings())
 
-	pNexUniqueIDInfo.WriteTo(rmcResponseStream)
+	pUniqueIDInfo.WriteTo(rmcResponseStream)
```

Comment:
Let's preserve the original name here

```suggestion
	pNexUniqueIDInfo.WriteTo(rmcResponseStream)
```